### PR TITLE
ARC-2106: Cortex supports srt-vtt transcription 

### PR DESCRIPTION
**Pull request by ademgcl**
Cortex transcribe now supports srt-vtt return
Youtube-dl now has patched version that works

### CODE REVIEW

File: helper_apps/MediaFileChunker/package.json [hunk 1]
```diff
@@ -18,6 +18,6 @@
     "ioredis": "^5.3.1",
     "public-ip": "^6.0.1",
     "uuid": "^9.0.0",
-    "ytdl-core": "git+ssh://git@github.com:khlevon/node-ytdl-core.git#v4.11.3-patch.1"
+    "ytdl-core": "github:GreepTheSheep/node-ytdl-core"
```
Comment (jmacAJ) [inline helper_apps/MediaFileChunker/package.json:21]: Probably don't want to take a dependency on this sort of unknown package - it's a security risk for sure.  Might be better for us to fork ytdl-core to aj-archipelago?

File: graphql/plugins/openAiWhisperPlugin.js [hunk 2]
```diff
@@ -14,11 +14,33 @@ import http from 'http';
...
 import url from 'url';
 import { promisify } from 'util';
+import subsrt from 'subsrt';
 const pipeline = promisify(stream.pipeline);
 
 
 const API_URL = config.get('whisperMediaApiUrl');
 
+function alignSubtitles(subtitles) {
+    let result = [];
```
Comment (jmacAJ) [inline graphql/plugins/openAiWhisperPlugin.js:24]: const?

**Review by jmacAJ — CHANGES_REQUESTED** [review]
I'd fix the dependency for sure.  Otherwise lgtm.

File: graphql/plugins/openAiWhisperPlugin.js (hunk 2 above)
Comment (ademgcl) [reply graphql/plugins/openAiWhisperPlugin.js:24]: forgotten from previous assignment

File: helper_apps/MediaFileChunker/package.json (hunk 1 above)
Comment (ademgcl) [reply helper_apps/MediaFileChunker/package.json:21]: so i found out the prev patcher that we were using patched it github:khlevon/node-ytdl-core#v4.11.4-patch.2 
i switched to it, does it make sense or should i still fork?

**Review by jmacAJ — APPROVED** [review]

**Pull request merged**